code/modules/antagonists/revenant/revenant_abilities.dm

```diff
@@ -28,6 +28,9 @@
 
 //Harvest; activated by clicking the target, will try to drain their essence.
 /mob/living/simple_animal/revenant/proc/Harvest(mob/living/carbon/human/target)
+	if(istype(get_turf(src), /turf/closed))
+		to_chat(src, span_revenwarning("You can't quite materialize in this space! Try a more open space.")
+		return
 	if(!castcheck(0))
```

Comment:
what does this proc do goof

Reply:
also here
![image](https://user-images.githubusercontent.com/53777086/151123229-c12c2c43-b9ac-4c83-b11d-83e1e8c1d646.png)